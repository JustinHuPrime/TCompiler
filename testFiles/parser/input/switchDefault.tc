module foo;

int bar(int i) {
  switch (i) {
    default: return 1;
  }
}