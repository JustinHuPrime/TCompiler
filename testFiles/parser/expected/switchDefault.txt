testFiles/parser/switchDefault.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/switchDefault.tc, 3, 1, int(int)))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, int), ID(3, 5, bar, REFERENCES(testFiles/parser/switchDefault.tc, 3, 1)), KEYWORDTYPE(3, 9, int), ID(3, 13, i, REFERENCES()), STAB(ENTRY(i, VARIABLE(testFiles/parser/switchDefault.tc, 3, 9, int))), COMPOUNDSTMT(3, 16, STAB(), SWITCHSTMT(4, 3, ID(4, 11, i, REFERENCES(testFiles/parser/switchDefault.tc, 3, 9)), SWITCHDEFAULT(5, 5, RETURNSTMT(5, 14, LITERAL(5, 21, UBYTE(1))), STAB())))))
